hazelcast/src/main/resources/hazelcast-full-example.yaml

```diff
@@ -1219,16 +1217,16 @@ hazelcast:
           eviction-policy: LFU
       wan-replication-ref:
         my-wan-cluster-batch:
-         merge-policy: com.hazelcast.map.merge.PassThroughMergePolicy
-         filters:
-           - com.example.SampleFilter
-           - com.example.SampleFilter2
-         republishing-enabled: false
+          merge-policy: com.hazelcast.map.merge.PassThroughMergePolicy
```

Comment:
The class `com.hazelcast.map.merge.PassThroughMergePolicy` does not exist any more.

Reply:
fixed.